components/nds-filters/src/FilterSummary.js

```diff
@@ -92,56 +87,52 @@ function ResultsFilters({ filters }) {
 		</ul>
 	);
 }
+function ResultsSorting({ sorting }) {
+	if (!sorting.length) return null;
 
-function ResultsSorting({ active, inactive }) {
-	if (!inactive.length && !active) return null;
-	return (
-		<p className="filter-summary__sort hide-print">
-			{active && (
-				<>
-					<span>
-						<span className="visually-hidden">Sorted by</span> {active.label}
-						<span className="visually-hidden">.</span>
-					</span>{" "}
-					|
-				</>
-			)}
+	// Find default active value if it exists
+	let defaultValue = "";
+	for (let i = 0; i < sorting.length; i++) {
+		if (sorting[i].active === true) {
+			defaultValue = sorting[i].value;
+			break;
+		}
+	}
+
+	// Run callback function passed in to the selected sorting option
+	const handleChange = e => {
+		const callback = sorting[e.target.selectedIndex].callback;
+		if (typeof callback === "function") {
+			callback();
+		}
+	};
+
+	// Hide button via JS, so it's visible if JS is disabled
+	const buttonStyle = { display: "none" };
 
-			{inactive.length &&
-				inactive.map(
-					(
-						{
-							label,
-							destination,
-							method,
-							elementType,
-							onClick,
-							className = ""
-						},
-						index
-					) => {
-						const ElementType = defineElementType(onClick, elementType);
-						const props = {
-							className: classnames([
-								"filter-summary__sort-control",
-								className
-							]),
-							"aria-label": `Sort by ${label}`,
-							[populateMethodProperty(onClick, method, ElementType)]: onClick
-								? onClick
-								: destination
-						};
-
-						return (
-							<span key={index}>
-								{" "}
-								<ElementType {...props}>{label}</ElementType>{" "}
-								{index + 1 < inactive.length && "|"}
-							</span>
-						);
-					}
-				)}
-		</p>
+	return (
+		<div className="filter-summary__sorting">
+			<label className="filter-summary__sort-label" htmlFor="sorting-options">
+				Sort by
+			</label>
+			<select
+				id="sorting-options"
+				onChange={handleChange}
+				defaultValue={defaultValue}
+				className="filter-summary__select"
+			>
+				{sorting.map(({ label, value }, index) => {
+					return (
+						<option value={value} key={index}>
```

Comment:
Don't we need to render a `selected` attribute here based on the `active` property?

Reply:
A better way is apparently to do what has been done here, i.e. set the `defaultValue` prop on the select element:

https://stackoverflow.com/a/24574465/1579944